.env.development

```diff
@@ -7,5 +7,7 @@
 ## `.env.development` stays in git with default or empty values
 
 MAPBOXGL_ACCESS_TOKEN=
-API_ENDPOINT=https://app-development.plant-for-the-planet.org
-CDN_URL=
+API_ENDPOINT=app-development.plant-for-the-planet.org
+AUTH0_CUSTOM_DOMAIN=pftp.eu.auth0.com
+AUTH0_CLIENT_ID=yxABVrSIbmSO6QeTfZjkJefDnW2vrE7y
```

Comment:
Is this an information we want to make public?

Reply:
I have remove it. these feature depends on the login that's why i had to branch off feature/login

Reply:
Ok, but as that branch is now already merged into develop, I would suggest you should rebase your branch to develop as well. (`git rebase develop`).